PhysicsTools/NanoAOD/plugins/EvaluateMuonMVAID.cc

```diff
@@ -0,0 +1,59 @@
+//
+//
+//
+//
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "DataFormats/PatCandidates/interface/Muon.h"
```

Comment:
I think most of the includes below are taken from the b-jet mva file and are not needed here? Could you please remove them (I also see some duplicates)?